cel/decls.go

```diff
@@ -564,6 +564,32 @@ func FunctionBinding(binding functions.FunctionOp) OverloadOpt {
 	}
 }
 
+// LocaleOptionBinding conditionally adds a new binding based on locale, passing newOp an additional final
+// argument that represents the configured locale as a string. If no locale was configured, LocaleOptionBinding
+// will do nothing. If no name/overloadID match is made, then LocaleOptionBinding will return an error.
+func LocaleOptionBinding(name, overloadID string, newOp functions.FunctionOp) EnvOption {
```

Comment:
Do you need this to be part of the top-level CEL API exposed to K8s? If not, it's probably fine to just have a local unexported helper that does this within the `strings.go`

Reply:
Also, the use is limited enough that you could just declare the three argument signature and proxy to a common implementation.

Reply:
`LocaleOptionBinding` interacts with the unexported `Env.locale` field, so I don't think we could move it to another package without making `locale` exported. We could definitely do that if you think it'd be better to turn it into `Env.Locale` though.

Reply:
I see, I suppose the other option would be to move the Locale to an option passed to `ext.Strings()` which would then allow you to store the parsed locale in to the library instance returned by `ext.Strings()` and make it clear that `Locale` only applies within the `Strings` library for now.

Reply:
(marking as resolved, `LocaleOptionBinding` has been removed and the locale option has been moved into `ext/strings.go`)